docs/ionic/authentication.md

```diff
@@ -0,0 +1,94 @@
+# STEP 1: install this plugin: 
```

Comment:
Use capitalization instead of uppercase for the word "step". I find that when something is placed in uppercase by itself too many times it can come off a bit aggressive.